### PR TITLE
WIP create si entities

### PR DESCRIPTION
**Pull request by pimg**

### CODE REVIEW

File: edocumentation-application/src/test/java/org/dcsa/edocumentation/controller/ShippingInstructionControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  @Autowired ShippingInstructionService shippingInstructionService;
+
+  @Autowired private TransactionTemplate transactionTemplate;
+
+  @Test
+  void test() {
+
+    EquipmentTO equipmentTO = EquipmentTO.builder().equipmentReference("eq_ref_01").build();
+
+    UtilizedTransportEquipmentTO utilizedTransportEquipmentTO = UtilizedTransportEquipmentTO.builder()
```
Comment (FinnAtAsseco) [inline edocumentation-application/src/test/java/org/dcsa/edocumentation/controller/ShippingInstructionControllerTest.java:28]: Some of this test data has data factories

File: edocumentation-service/src/main/java/org/dcsa/edocumentation/service/EquipmentService.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class EquipmentService extends EnsureResolvable<EquipmentTO, Equipment> {
+  private final EquipmentRepository equipmentRepository;
+  private final EquipmentMapper equipmentMapper;
+
+  private final ExampleMatcher exampleMatcher =
+      ExampleMatcher.matchingAll().withIncludeNullValues().withIgnorePaths("equipmentReference");
```
Comment (FinnAtAsseco) [inline edocumentation-service/src/main/java/org/dcsa/edocumentation/service/EquipmentService.java:29]: Isn't equipmentReference exactly what you want to look for, but you ignore it instead?

**Review by nt-as — APPROVED** [review]

File: edocumentation-application/src/test/java/org/dcsa/edocumentation/controller/ShippingInstructionControllerTest.java (hunk 1 above)
Comment (pimg) [reply edocumentation-application/src/test/java/org/dcsa/edocumentation/controller/ShippingInstructionControllerTest.java:28]: this test will ultimately be removed when the controller is in place

File: edocumentation-service/src/main/java/org/dcsa/edocumentation/service/EquipmentService.java (hunk 2 above)
Comment (pimg) [reply edocumentation-service/src/main/java/org/dcsa/edocumentation/service/EquipmentService.java:29]: after this merge we will look into consolidating this with your PR's equipment service and resolve it that way

**Pull request merged**